src/templates/EditorialNewsletter/email/Header.js

```diff
@@ -8,33 +27,59 @@ export default ({ meta }) => {
     format && format.indexOf('format-covid-19-uhr-newsletter') !== -1
 
   return (
-    <tr>
-      <td
-        align='center'
-        valign='top'
-        style={{ borderBottom: `1px solid ${colors.divider}` }}
-      >
-        <a
-          href={`https://www.republik.ch${path ? path : `/${slug}`}`}
-          title='Im Web lesen'
+    <>
+      <tr>
+        <td
+          align='center'
+          valign='top'
+          style={{ borderBottom: !isCovid19 && `1px solid ${colors.divider}` }}
```

Comment:
shouldn't pass false to react style afaik
```suggestion
          style={{ borderBottom: !isCovid19 ? `1px solid ${colors.divider}` : undefined }}
```

Reply:
Get it. Thanks.